ONAV-973: Hardware kit checklists

jhiggins-cpr opened the pull request.

jhiggins-cpr committed:
First draft of updated hardware kit requirements

jhiggins-cpr committed:
Add backpack software checks

jhiggins-cpr committed:
Fix formatting

rhysfaultless-cpr commented:
Content looks good. I plan to format these files with Prettier before merging this PR.

Commit:
Fixed 3 spelling typos.

Adjusted source line spacing, no affect to rendered site.

Ran Prettier to adjust source table formatting.
